.travis.yml

```diff
@@ -13,3 +14,20 @@ addons:
 
 before_script:
     - TEST=true npm run build
+
+
+deploy: #update translation strings on transifex
+    provider: script
+    skip_cleanup: true
+    script:
+        - npm run build
+        - npm run i18n
+        - pip install virtualenv
+        - virtualenv ~/env
+        - source ~/env/bin/activate
+        - pip install transifex-client
+        - sudo echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"$TRANSIFEX_API_USER"$'\npassword = '"$TRANSIFEX_API_KEY"$'\n' > ~/.transifexrc
```

Comment:
We should check a lightweight way to make the job, maybe check TransifleX api and call it with system call (curl) or simple script

Reply:
It looks good to me. Maybe we could use `sudo pip install` directly (without the virtualenv), but that's a detail.